notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>use <code></code> in latex instead of " on the left</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>